### DISCUSSION TITLE
多表单字段联动

### DISCUSSION THREAD

**Discussion by PlutoCA** [alibaba/formily#2412] [Q&A 2.x] [answered]
场景： A表单的type 控制B表单字段显示隐藏

如何在 B表单通过dependencies 拿到 A表单的值

https://codesandbox.io/s/hungry-ramanujan-150ui?file=/App.tsx

目前我是通过`onFieldChange` 监听A表单的字段 再通过 B表单的scope 传给B表单。

想请教一下康康有没有更好的的方法。
谢谢！

**Comment by janryWang** [comment, marked as answer]
文档里有的，都是现成的

**Reply by zaxtseng** [reply]
同样的需求,请问可以贴出文档中满足这个条件的文档地址吗?翻看了很多,没有找到

**Comment by chqa** [comment]
有个简单的办法  eventBus
